deploy/scripts/deploy-docker.sh

```diff
@@ -3,26 +3,28 @@
 DIR_PATH=$(pwd)
 CONF_FILES_DIR="conf-files"
 
-IMAGE_NAME="ufcglsd/iguassu"
+IMAGE_NAME="ufcglsd/iguassu:latest"
 CONTAINER_NAME="iguassu"
```

Comment:
In the near future, we'll use various docker containers/images, only one per application component, (e.g. one container to the dashboard, other to the backend). I find it useful to leave these fields as configurable. Move the container/images definitions to a conf file.